test/parse-error-test/annotation_semantics/parse-error.expected

```diff
@@ -0,0 +1,4 @@
+[error] test/parse-error-test/annotation_semantics/input.ddl[3.4] failure: string matching regex `\p{javaJavaIdentifierStart}\p{javaJavaIdentifierPart}*' expected but `)' found
```

Comment:
I think I figured out the problem. This can't be parsed as an inferenceRule, so the parser tries to parse it as other types of rule and fails. I tried `~!` and it didn't work, as it's preventing backtracking in the sequential composition, but not the upper level. I'm still looking for a good solution here.